client.go

```diff
@@ -627,6 +630,19 @@ func (c *client) DesiredLRPSchedulingInfos(logger lager.Logger, traceID string,
 	return response.DesiredLrpSchedulingInfos, response.Error.ToError()
 }
 
+func (c *client) DesiredLRPSchedulingInfoByProcessGuid(logger lager.Logger, traceID string, processGuid string) (*models.DesiredLRPSchedulingInfo, error) {
+	request := models.DesiredLRPByProcessGuidRequest{
```

Comment:
Here I am reusing the DesiredLRPByProcessGuidRequest model since it is the same. Not sure if the new endpoint should have a dedicated model even if they match.

Reply:
To me it seems acceptable to reuse the response structure if they mach